agent/test-agent-cli/design/RUNBOOK.md

```diff
@@ -0,0 +1,99 @@
+# Steps to use Test-agent Command Line Interface
+`test-agent-cli` lets you write Testsys test using Bash and help in receiving and sending information from/to the Testsys cluster.
```

Comment:
It seems like we use "TestSys" instead of "Testsys" as the proper capitalization.